LeanplumSDK/LeanplumSDK/Classes/Internal/Leanplum.m

```diff
@@ -2249,6 +2249,14 @@ + (void)setTrafficSourceInfoInternal:(NSDictionary *)info
     [[LPRequestSender sharedInstance] send:request];
 }
 
++ (void)setLocale:(NSLocale *)locale
+{
```

Comment:
What is the use case - would this be called before starting Leanplum or at some point after it started?
If it has started, the locale in the start API call will be the system one.

Reply:
The idea was for calling it after Leanplum is started, in order to simplify a bit complex variations of starting.

If you think it is better to move it to start, and/or to add some stored property for overriding the system's locale, so the start would take that one, I'm ok with it as well.

Reply:
Improved in a way that `setLocale` now can be called both before and after Leanplum is started